### PR TITLE
Wizard workflow

### PR DESCRIPTION
**Pull request by k8reindeer**
## Links
* Issue link https://github.com/usdigitalresponse/neighbor-express/issues/121
* Documentation links

## GIF/Screenshots:
image
image
image

## Changes:
* There are 2 steps to sending emails (enqueueing and then sending). Now these are separated 

## How To Test:
* Follow local development instructions in send_email_messages/README.md.
_or_ I have deployed this code live to staging

## Feedback I'm looking for:
UI feedback, any cleanup we want to do, does this breakdown of steps make sense, anything else that we could do to make this error-protective and keep users on the right path.

## Things left to do before deploying:
- [ ] ...

### CODE REVIEW

File: send_email_messages/frontend/refresh-queue.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    setWarnings([]);
+    let {messagesToCreate, warnings} = await computeMessagesToCreate();
+    setWarnings(warnings);
+
+    if (messagesToCreate.length == 0) {
+      setResult(`No new messages to enqueue.`);
+      return;
+    }
+    if (messagesToCreate.length > 50) {
+      addWarning(`You can only create 50 emails at once, but we found ${messagesToCreate.length} Doing the first 50 now... remember to do this again to get the rest!`);
```
Comment (keithk) [inline send_email_messages/frontend/refresh-queue.js]: I wonder if this error message could be more clear?

**Review by keithk — APPROVED** [review]
A very tiny question on an error message, but otherwise I think this looks great. 

The `computeMessagesToCreate` function could be a good candidate for breaking down, since it has a few nested loops, but isn't too complex to read through now.

File: send_email_messages/frontend/refresh-queue.js (hunk 1 above)
Comment (k8reindeer) [reply send_email_messages/frontend/refresh-queue.js]: Removed the need for this error message -- this is something we can just prevent from ever happening by batching, which is a better user experience than having to see this error and understand what to do

**Comment by k8reindeer** [comment]
Yeah, I imagine `computeMessagesToCreate` will be refactored as part of implementing https://github.com/usdigitalresponse/neighbor-express/issues/122 :) My idea would be to break it down so it loops over message types, and queries for deliveries in the relevant status for that message type (where now it loops over deliveries, and then over message types in the inner loop). But yeah, not going to do that in this PR.

**Comment by k8reindeer** [comment]
(manually released the block to staging, walnut creek and paterson)

**Pull request merged**